src/fastapi_zitadel_auth/testing/utils.py

```diff
@@ -136,28 +139,59 @@ def create_test_token(
     expired: bool = False,
     invalid_iss: bool = False,
     invalid_aud: bool = False,
-    scopes: str = "scope1",
+    scopes: str = "openid profile",
     evil: bool = False,
-    role: str | None = None,
+    role: Optional[str] = None,
```

Comment:
please do not import `typing` helpers like `Optional`, `Dict` etc anywhere -- since Python 3.10 which is the current minimum + PEP604 we can use the built-in approach: 

```python
role: str | None = None
```